test/mock-server/Dockerfile

```diff
@@ -8,6 +8,6 @@ USER deno
 
 COPY *.ts *.pem lock.json /app/
 COPY /frontend/* /app/frontend/
-RUN deno cache deps.ts --lock=lock.json
+RUN deno cache /app/deps.ts --lock=lock.json
```

Comment:
Think I'm missing something here, but why is this change needed?

Reply:
It was not able to find `deps.ts` without adding without specifying the home directory

Reply:
Actually, either way is correct because of:

```Dockerfile
COPY *.ts *.pem lock.json /app/
```

which copied all `*.ts` files (including `deps.ts`) into `/app`. But because `/app` was already the `WORKDIR`, so it should work even without `/app` in `deps.ts` path.

As you can see in the last line:

```Dockerfile
CMD ["run", "--allow-net", "--allow-read", "--allow-write", "server.ts"]
```

there's no `/app` at all

Reply:
Cool. I've changed it